en/02_Developer_Guides/08_Performance/07_Read_only_database_replicas.md

```diff
@@ -25,7 +25,9 @@ SS_DATABASE_USERNAME_REPLICA_01="my-replica-user"
 SS_DATABASE_PASSWORD_REPLICA_01="my-replica-password"
 ```
 
-Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. They are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
+Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. Those are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
+
+The `SS_DATABASE_SERVER`, `SS_DATABASE_USERNAME`, and `SS_DATABASE_PASSWORD` must be defined for each replica. If a port other than the default should be used, `SS_DATABASE_PORT` must also be defined for each replica.
```

Comment:
Added this because most of the text talks about `SS_DATABASE_SERVER` - and people might intuitively think if the same username and password is used across the board, they don't have to redefine them.